### PR TITLE
Writer

### PR DESCRIPTION
**Pull request by BPRateiro**
- Refactored the prototyping notebook into multiple scripts based on the 'quality' of the tables
- Wrote `models.py` containing the tables in ORM form
- Wrote `main.py` that writes these tables to the database in an orderly fashion

Most modifications are concentrated in both `models.py` and `main.py`

### CODE REVIEW

File: src/analytics.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                              index=df.index).iloc[:, :4]
+
+    # Set index to integer
+    df.index = df.index.astype('int64', copy=False)
+
+    return df[['tier', 'wins', 'copies']]
+
+
+def get_analytics_games(df):
+    "Returns data frame 'analytics_games'"
```
Comment (eliassoares) [inline src/analytics.py:37]: Normally we use triple `"` to docstring.

File: src/main.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
+import os
+
+from sqlalchemy_utils import database_exists, create_database
+from sqlalchemy import create_engine, update, select
+from sqlalchemy.engine import URL
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.sql import func
+
+from models import Set, AnalyticsWins, drop_all, create_all
```
Comment (eliassoares) [inline src/main.py:9]: `drop_all` is not used.

File: src/main.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+from analytics import get_analytics
+from card import get_card_information
+from raw import request_active
+
+import pandas as pd
+
+
+def start_engine():
```
Comment (eliassoares) [inline src/main.py:17]: I guess `get_engine` can be more legible.

**Review by eliassoares — APPROVED** [review]
Excellent work!
I made some comments, but nothing that can block the merge.
lgtm

File: src/analytics.py (hunk 1 above)
Comment (BPRateiro) [reply src/analytics.py:37]: Fair enough!

File: src/main.py (hunk 2 above)
Comment (BPRateiro) [reply src/main.py:9]: Removed from imports.

**Pull request merged**